### ISSUE TITLE
N64 freezes after 2nd Save State

### ISSUE THREAD

**Issue by bertomuggs** [OpenEmu/OpenEmu#4090]
When I play any N64 game and save state for the second time OpenEmu freezes. I have to force quit and reopen the app. I tested this with multiple games and it freezes with all of them. Tried Super Nintendo, no freezing. Works fine. 

I was on 2.2 so I installed 2.1.1 by just dragging into applications folder then telling the system to replace the 2.2 file. Tried everything again and it still freezes. 

Running macOSX 10.15.2

It freezes with or without my PS4 controller plugged in.

Also, this didnt happen before I updated to 2.2 today. So maybe I need to do a more thorough down grade than just dragging the app into the applications folder. Any ideas?

**Comment by clobber** [comment]
No a downgrade won't help you, in fact it will make things worse. N64 emulation overall has compatibility bugs -- Mupen64Plus (the core plugin) and GLideN64 (the GPU plugin) are still in a volatile work-in-progress state which means it is not recommended to rely on save states as those can break between core plugin updates. While I cannot replicate your issue, I do not doubt it is happening. Instead, rely on in-game battery saves if available.

To get your N64 games in working order, load the games without restoring an autosave state on first load:

* Go to OpenEmu > Preferences > Library, click "Reset warnings"
* Open your game and click "No" if prompted to "Continue where you left off"

**Comment by bertomuggs** [comment]
Clobber actually you are wrong. I decided to uninstall OpenEmu. Made sure and wiped every aspect of it from the whole entire file system. I kept my save states. I kept my ROMs. Everything else got wiped out. Then I reinstalled the previous version, put back my save states and my Rom. Started it back up and I could save as many times I want without the game freezing up.

So maybe revise your answer to reflect what I did so this thread can actually help other people.
